test/metabase/test/data/generic_sql.clj

```diff
@@ -130,6 +146,38 @@
             (qualify+quote-name driver database-name dest-table-name)
             (quot (pk-field-name driver)))))
 
+(defn standard-inline-column-comment-sql
+  "Generic inline COMMENT that driver can mixin if supported."
+  [_ field-comment]
+  (if field-comment
+    (format "COMMENT '%s'" field-comment)
+    (str "")))
```

Comment:
`(str "")` is redundant. You can just do `""`. It would probably make more sense instead to just have this function return `nil` instead of a blank string anyway

Reply:
e.g.

```clojure
(when (seq field-comment)
   (format "COMMENT '%s'" field-comment))
```

Note the use of `seq` above, which is called nil-punning, to make sure `field-comment` is non-blank (`(seq "")` is `nil`).